docs/sources/user_guide/runtime_assurance.rst

```diff
@@ -216,3 +217,21 @@ The mathematical formulations used by the RAM rely on an implicit assumption abo
   There exist a fundamental trade-off between conservatism and optimality at the control level. The more certainty one has in things like dynamics, localization, and communication timing, the less conservatism must be introduced to ensure a low probability of collision.
 
   The Supervisor is able to account for uncertainty in an explicit and quantitative way. Please `contact 3laws`_ to learn more about the process of tailoring Supervisor to better handle uncertainties and communication delays in your system.
+
+
+Visualization
+=============
+
+The RAM publishes a set of markers to visualize the processing done by the Safety Maps, and the safety constraints generated by the Kernel Generator:
+
+.. image:: ../data/ram_viz.png
+  :width: 800px
+  :align: center
+  :alt: RAM visualization
+
+|
+
+This data is published on the **/lll/ram/markers** topic. The :ref:`control panel <control_panel_viz>` can provide you with a an rviz configuration file to visualize this data.
```

Comment:
I prefer the style of academic papers where the language does not involve "you" or "we".  In this case, I don't believe there is a loss to say ".The control panel provides and rviz configuration file to visualize this data." To me, it is more professional.  However, that content already exists in the control_panel.rst file; so is redundant here.

Reply:
Fixed the `you`. Not redundant though, control panel talks about the existence of an RViz config file, whereas the RAM section explains (for now succinctly) what is vizualised.